public/404/404.php

```diff
@@ -0,0 +1,28 @@
+<?php require_once("../../private/initialize.php"); ?>
+
+<?php
+    
+    $page_title = "Yabe | Browse | Products";
+    $style_sheets = [
+        "/css/common.css",
+        "/css/404.css",
+    ];
+    $scripts = [
+        "/js/common.js",
+    ];
+    
+    include(SHARED_PATH . "/top.php");
+
+?>
+    <body>
+    <div class="container">
+        <div class="first-4">4</div>
```

Comment:
Please make the two 4s bolder so that they match the line weight of the logo in the middle. It's one of typography basic rules: Keep the font-weight consistent.

Also I think coloring them black would fit the logo better.